src/server/pfs/server/driver.go

```diff
@@ -1045,6 +1042,9 @@ func (d *driver) inspectCommit(ctx context.Context, commit *pfs.Commit, block bo
 						return fmt.Errorf("Unmarshal: %v", err)
 					}
 				case watch.EventDelete:
+					// Note: this error is detected explicity in
+					// src/server/worker/master.go. If this error text is changed, that
+					// file should be changed too
```

Comment:
We have some codified pfs errors in `src/server/pfs/pfs.go` you could just add `ErrCommitDeleted` there to enforce that those to use the same error texts.